src/lib/journey/callbacks/idp/idp.mock.ts

```diff
@@ -1,47 +0,0 @@
-export const docsExample = {
```

Comment:
I removed this since this type of callback is what is returned for mobile devices, instead of web. For web apps the "Select IDP" callback is returned.